schemas/2015-08-01/Microsoft.Cache.json

```diff
@@ -0,0 +1,157 @@
+﻿{
+    "id": "http://schema.management.azure.com/schemas/2015-08-01/Microsoft.Cache.json",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "Microsoft.Cache",
+    "description": "Microsoft Cache Resource Types",
+    "definitions": {
+        "virtualNetworkId": {
+            "type": "string",
+            "pattern": "^/subscriptions/[^/]*/resourceGroups/[^/]*/providers/Microsoft.ClassicNetwork/virtualNetworks/[^/]*$",
+            "description": "The resource id of an Azure Classic Virtual Network"
+        },
+        "ipV4Address": { 
+            "type": "string",
+            "pattern": "^\\d+\\.\\d+\\.\\d+\\.\\d+$",
+            "description": "An IP address in a Virtual Network"
+        },
+        "Redis": {
+            "type": "object",
+            "properties": {
+                "type": {
+                    "enum": [
+                        "Microsoft.Cache/Redis"
+                    ]
+                },
+                "apiVersion": {
+                    "enum": [
+                        "2015-08-01"
+                    ]
+                },
+                "properties": {
+                    "type": "object",
+                    "properties": {
+                        "sku": {
+                            "type": "object",
+                            "properties": {
+                                "name": {
+                                    "oneOf": [
```

Comment:
Correct - the default just lets JSON editors suggest a default for people who are authoring a template using your schema. The default value won't end up in the deployment unless the user chooses that default in their schema. 
For an example, see how these other RP schemas specify defaults: Microsoft.AppService.json, Microsoft.Scheduler.json, Microsoft.Cache.json.